runtime/plugin/man.vim

```diff
@@ -1,6 +1,21 @@
-if get(g:, 'loaded_man', 0)
+if exists('g:loaded_man')
   finish
 endif
 let g:loaded_man = 1
 
-command! -count=0 -nargs=+ Man call man#get_page(<count>, <f-args>)
+if !exists("g:find_man_window")
+  let g:find_man_window = 1
+endif
+
+if !exists('g:man_synopsis')
+  let g:man_synopsis = 'SYNOPSIS'
+endif
+
+command! -bang -complete=customlist,man#Complete -nargs=* Man call
+      \ man#get_page(<bang>0, 'edit', <f-args>)
+command! -bang -complete=customlist,man#Complete -nargs=* Sman call
+      \ man#get_page(<bang>0, 'split', <f-args>)
+command! -bang -complete=customlist,man#Complete -nargs=* Vman call
+      \ man#get_page(<bang>0, 'vsplit', <f-args>)
+command! -bang -complete=customlist,man#Complete -nargs=* Tman call
+      \ man#get_page(<bang>0, 'tabe', <f-args>)
```

Comment:
Adding new commands should not be done lightly, especially for a default plugin. E.g. if user has `:Tmux` command, now tab-complete will complete `Tman` first. This drives people crazy.

Let's hold off on the new commands and leave the question of how to provide these alternate modes for another PR.